### ISSUE TITLE
Midfoot arthritis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11220]
**Preferred term label:**
Midfoot arthritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP, oblique, or lateral view x-ray of the foot showing radiographic evidence of osteoarthritis in the midfoot (tarsometatarsal (TMT) joints, naviculocuneiform joint, intercuneiform joints, and/or metatarsal cuneiform joints), such as joint space narrowing, subchondral sclerosis, subchondral cysts, and/or osteophyte formation in the tibiotalar joint

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Midfoot arthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001333
- label: Midfoot arthritis
- parents: [Abnormal metatarsal morphology (HP:0001832)](https://hpo.jax.org/app/browse/term/HP:0001832)
- pmid: PMID:32405198
- def: AP, oblique, or lateral view x-ray of the foot showing radiographic evidence of osteoarthritis in the midfoot (tarsometatarsal (TMT) joints, naviculocuneiform joint, intercuneiform joints, and/or metatarsal cuneiform joints), such as joint space narrowing, subchondral sclerosis, subchondral cysts, and/or osteophyte formation in the tibiotalar joint.
- comment: Midfoot includes Tarso-metatarsal (TMT) joints and naviculo-cuneiform joints (NCJ). Talo-navicular joint (TNJ) and calcaneocuboid joint (CCJ) are considered to be part of hindfoot along with sub-talar joint. Tarso-metatarsal joints are divided into three columns anatomically: the medial column (First Tarsometatarsal joint), the middle column (second and third TMT joints) and the lateral column (Fourth and Fifth TMT or Metatarsocuboid joints). TMT joints are collectively known as Lisfranc joints as well.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11220
- synonyms:
